bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/FileCollection.java

```diff
@@ -43,34 +42,20 @@ public FileCollection(Supplier<Path> base, PassageFileExtension extension) {
 	}
 
 	public Collection<Path> get() throws LicensingException {
-		HunterFiles hunter = new HunterFiles(extensions);
-		try {
-			Files.walkFileTree(base.get(), hunter);
+		try (Stream<Path> files = filesIn(base.get())) {
+			return files(files);
```

Comment:
It does not read well. Should it rather be `return filtered(all);`?

Reply:
Sounds good.